src/main/java/edu/hw1/Task1.java

```diff
@@ -0,0 +1,34 @@
+package edu.hw1;
+
+import java.util.logging.Logger;
+
+public class Task1 {
+
+//    public static void main(String[] args) {}
+
+    private final static Logger LOGGER = Logger.getLogger("LOGGER");
```

Comment:
Есть душная специфкация, что правильно писать
```suggestion
    private static final Logger LOGGER = Logger.getLogger("LOGGER");
```